src/webui/components/reaction/Delete.tsx

```diff
@@ -0,0 +1,13 @@
+
+import { ClickArea, Icon } from '^/webui/designsystem/module';
+
+export type Props = {
+    deleteHandler: () => void;
+};
+
+export default function Component({ deleteHandler }: Props)
+{
+    return <ClickArea clickHandler={deleteHandler}>
```

Comment:
As discussed, we need to split this up into a `DeleteIcon` and a `DeleteButton`.